app/documents.js

```diff
@@ -163,6 +163,7 @@ export class Document extends Base {
 
     spinner.start();
     const delay = (duration) => new Promise((resolve) => setTimeout(resolve, duration));
+    var built_docs_map = {};
```

Comment:
use `const` instead of `var` since you aren't changing it.

Reply:
need `var` to rest this variable back to empty object in the then block